src/layer/vector/Circle.js

```diff
@@ -24,6 +24,13 @@ import {Earth} from '../../geo/crs/CRS.Earth';
 
 export var Circle = CircleMarker.extend({
 
+	options: {
+		// @option title: String= 'Circle'
```

Comment:
We should consider not using default titles.

Relatedly, I've been thinking of partially reverting https://github.com/Leaflet/Leaflet/pull/7895 to remove the default `alt="Marker"`, for several reasons (passing accessibility audits by default doesn't encourage unique titles, localization, inconsistent names depending on if `title` or `alt`/`desc` was used). But we could open a new issue to discuss both.

Reply:
I added defaults primarily for consistency with the Marker. I can remove defaults here, and we can open a separate issue to discuss `alt` for the Marker. And depending on the discussion there, we can either remove the default for Marker or add for the vector types.

Reply:
Eight minutes passed, and my opinion has changed 🙃 
Let's leave defaults as is and discuss `alt` for Marker and default `title`s for vectors in a separate issue. I can see the benefits of both providing defaults and not. I also think that consistency is important here. So discussing them together is a good way forward.

Reply:
Yep, we can always change this later. Opened https://github.com/Leaflet/Leaflet/issues/8267 as a reminder.